android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -227,6 +227,22 @@ public void onServiceDisconnected(ComponentName name) {
         customTabsSupported = bindService(customTabIntent, customTabsServiceConnection, Context.BIND_AUTO_CREATE);
     }
 
+    private void setUserProfileMenuItem(){
+        MenuItem userProfileMenuItem = navigationView.getMenu().findItem(R.id.nav_user_profile);
+        if (AuthUtil.isUserLoggedIn()) {
+            String email = SharedPreferencesUtil.getString(ConstantStrings.USER_EMAIL, null);
+            String firstName = SharedPreferencesUtil.getString(ConstantStrings.USER_FIRST_NAME, null);
+            String lastName = SharedPreferencesUtil.getString(ConstantStrings.USER_LAST_NAME, null);
+
+            if (firstName != null && lastName != null && !firstName.isEmpty() && !lastName.isEmpty())
+                userProfileMenuItem.setTitle(firstName + " " + lastName);
+            else if (firstName != null && !firstName.isEmpty())
+                userProfileMenuItem.setTitle(firstName);
```

Comment:
I think, you can use `TextUtils.isEmpty` for firstName and then just set text as `firstName + lastName`